docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -78,13 +79,14 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
-    The scheduler applies rolling updates as follows:
+    The scheduler applies rolling updates as follows by default:
 
-    * Stop the initial number of tasks according to `--update-parallelism`.
-    * Schedule updates for the stopped tasks.
-    * Start the containers for the updated tasks.
-    * After an update to a task completes, wait for the specified delay
-    period before stopping the next task.
+    * Stop the first task.
+    * Schedule updates for the stopped task.
+    * Start the containers for the updated task.
+    * If the update to a task returns `RUNNING`', wait for the
+    specified delay period then stopping the next task.
+    * If an update to a task returns `FAILED`, pause the update.
```

Comment:
Note that this decision to pause or continue isn't made on a task-by-task basis. If the first updated task fails later on while it's updating the 3rd task, that will still pause the update.